pilot/pkg/model/config_test.go

```diff
@@ -642,6 +664,44 @@ func TestAuthenticationPolicyConfig(t *testing.T) {
 				t.Errorf("AutheticationPolicy(%s:%d) => expected %#v but got %#v",
 					testCase.hostname, testCase.port, expected, out)
 			}
+
+			// Verify jwks_uri is got correctly through mock openID discovery server if not set in the auth policy.
+			if testCase.expected == "mockopenidonce" || testCase.expected == "mockopenidtwice" {
```

Comment:
As mentioned above, rewrite this test to avoid having this complicated code in verifying part.

Reply:
moved test to auth_cache_test.go and reverted change on this file.